config.yml

```diff
@@ -1,5 +1,5 @@
 ---
-docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+docker_hub_namespace: &docker_hub_namespace mallear/docker-buildbox
```

Comment:
?

Reply:
I had to push it to my repo for first use, forgot to revert the change

Reply:
Change reverted